### PR TITLE
9.18 Release notes

### PR DESCRIPTION
**Pull request by lzdanski**

### CODE REVIEW

File: astro/runtime-release-notes.md [hunk 1]
```diff
@@ -400,6 +400,17 @@ For more information about the major changes in this release, see the [Airflow b
 - Fixed an issue in Astro where logging features could be disrupted if you set `AZURE_CLIENT_ID` as an environment variable.
 - Fixed an issue where Astro audit logs listed a user's name as `User` for trigger events instead of their IDs.
 
+## Astro Runtime 9.18.0
+
+- Release date: August 14, 2024
```
Comment (utkarsharma2) [inline astro/runtime-release-notes.md:405]:
```suggestion
- Release date: August 15, 2024
```

**Review by utkarsharma2 — APPROVED** [review]

File: astro/runtime-release-notes.md [hunk 2]
```diff
@@ -400,6 +400,17 @@ For more information about the major changes in this release, see the [Airflow b
...
 - Fixed an issue where Astro audit logs listed a user's name as `User` for trigger events instead of their IDs.
 
+## Astro Runtime 9.18.0
+
+- Release date: August 15, 2024
+- Airflow version: 2.7.3
+
+### Additional improvements
+
+- If the `Taskinstance` state is `skipped`, also skip checking `subdaglist` ([#40578])(https://github.com/apache/airflow/pull/40578)
```
Comment (TJaniF) [inline astro/runtime-release-notes.md:410]:
```suggestion
- If the `Taskinstance` state is `skipped`, also skip checking `subdaglist` ([#40578](https://github.com/apache/airflow/pull/40578))
```

File: astro/runtime-release-notes.md [hunk 3]
```diff
@@ -400,6 +400,17 @@ For more information about the major changes in this release, see the [Airflow b
...
+## Astro Runtime 9.18.0
+
+- Release date: August 15, 2024
+- Airflow version: 2.7.3
+
+### Additional improvements
+
+- If the `Taskinstance` state is `skipped`, also skip checking `subdaglist` ([#40578](https://github.com/apache/airflow/pull/40578))
+- Added validation for the project URL that comes from installed providers, before displaying the URL in views ([#40933](https://github.com/apache/airflow/pull/40933))
+- Upgraded the minor and patch versions of some open-source provider packages. See [Astro Runtime 9.18.0 provider packages](https://www.astronomer.io/docs/astro/runtime-provider-reference#astro-runtime-9180)
```
Comment (TJaniF) [inline astro/runtime-release-notes.md:412]:
```suggestion
- Upgraded the minor and patch versions of some open-source provider packages. See [Astro Runtime 9.18.0 provider packages](https://www.astronomer.io/docs/astro/runtime-provider-reference#astro-runtime-9180).
```

**Review by TJaniF — APPROVED** [review]
LGTM :) just formatting things

**Pull request merged**